src/pods/edit-realtion/map-realtion-formvm.business.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { afterEach, vi } from 'vitest';
```

Comment:
file name let's change it to map-relation

I think nobody is updating this pod so rename as well the folder to

edit-relation

Reply:
and file name should be 

map-relation-form.vm.spec.ts

Reply:
It's fixed